frontend/src/views/event/DayOverview.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <side-bar>
+    <card-view>
+      <v-subheader class="text-uppercase subtitle-2">
+        Tagesübersicht
+      </v-subheader>
+      <v-skeleton-loader v-if="eventInstances.loading" class="ma-3"
+                         type="list-item@6" />
+      <picasso v-else
+               :camp="period.camp"
+               :event-instances="eventInstances.items"
+               :start="startOfDay"
+               interval-height="36"
```

Comment:
```suggestion
               :interval-height="36"
```
Otherwise, "36" is passed in as string (should be a number).